lib/testing-helpcenter/config.rb

```diff
@@ -16,6 +16,10 @@ def initialize(params = {})
       @browser = params.fetch(:browser, :chrome)
     end
 
+    def test_plan_name
+      "Help center domain: #{@server}, Version: #{Time.new.strftime('%d/%m/%Y')}"
```

Comment:
Extract `Time.new.strftime('%d/%m/%Y')` to separate method `def version` too

Reply:
Maybe it's better to return `def test_plan_name` to test_manager because it isn't similar to config
And move to config only `def version` method?

Reply:
@olga-nesterova Yeah, I agree